layouts/community/ergodox/dvorak_programmer_swe/keymap.c

```diff
@@ -203,6 +203,10 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
 ),
 };
 
+const uint16_t PROGMEM fn_actions[] = {
```

Comment:
Could you remove the `fn_actions` block?  All of it's features have been added as keycodes, and been deprecated in favor of `process_record_user`.


`ACTION_LAYER_TAP_TOGGLE(SYMB)` can be replaced `TT(SYMB)` in the keymap.

Reply:
Ok will remove that block.

But when I replace TT(SYMB) with ACTION_LAYER_TAP_TOGGLE(SYMB) that button do not work as wanted, instead of a layer tap toggle I get the text:  21f0

Do you know why? 
Is TT(SYMB) deprecated?

Reply:
`TT()` is not deprecated, `fn_actions` is. If you had `F(1)` in your keymap you would simply replace it with `TT(SYMB)`. However since it has already been changed this `fn_actions` block is not used anyway, so all you have to do is remove it.

Reply:
Ok. It is removed and pushed. Thanks!